src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py

```diff
@@ -75,3 +85,118 @@ def load_arguments(self, _):
 
     with self.argument_context('cosmosdb database') as c:
         c.argument('throughput', type=int, help='Offer Throughput (RU/s)')
+
+# SQL database
+    with self.argument_context('cosmosdb sql database') as c:
+        c.argument('account_name', options_list=['--account', '-a'], help="Cosmosdb account name", id_part=None)
+        c.argument('database_name', options_list=['--database', '-n'], help="Database name")
+        c.argument('throughput', help='Offer Throughput (RU/s). Default value is 400')
+
+# SQL container
+    with self.argument_context('cosmosdb sql container') as c:
+        c.argument('account_name', options_list=['--account', '-a'], help="Cosmosdb account name", id_part=None)
+        c.argument('database_name', options_list=['--database', '-d'], help="Database name")
+        c.argument('container_name', options_list=['--container', '-n'], help="Container name")
+        c.argument('partition_key_path', options_list=['--part'], help='Partition Key Path, e.g., \'/properties/name\'')
```

Comment:
1. options_list=['--partition-key-path', '-p']
2. Please give a more detailed description for help message.